app-devel/gcc+cross-mips64r6el/autobuild/build

```diff
@@ -0,0 +1,14 @@
+unset CFLAGS CXXFLAGS LDFLAGS CPPFLAGS
+
+mkdir -p build
+cd build
+AR=ar ../configure --prefix=/opt/abcross/mips64r6el --target=mipsisa64r6el-aosc-linux-gnuabi64 \
+	--with-sysroot=/var/ab/cross-root/mips64r6el --enable-shared --disable-multilib \
+	--enable-c99 --enable-long-long --enable-threads=posix --enable-languages=c,c++,fortran,lto \
+	--enable-__cxa_atexit --disable-altivec --disable-fixed-point --with-arch=mips64r6 --with-tune=mips64r6 \
```

Comment:
Altivec is only valid for power architecture and is irrelavant for mips64r6. Please remove `--disable-altivec`